api/krusty/diamondpatchref_test.go

```diff
@@ -0,0 +1,204 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package krusty_test
+
+import (
+	"testing"
+
+	kusttest_test "sigs.k8s.io/kustomize/api/testutils/kusttest"
+)
+
+func TestNamePrefixSuffixPatch(t *testing.T) {
```

Comment:
Need a new name, e.g. `TestConfMapNameResolutionInDiamondWithPatches`

Reply:
Ah yes, that explains the test failure here- I only tested the one file in isolation locally. Thanks :+1: